contrib/!lang/html/packages.el

```diff
@@ -197,10 +203,21 @@
   (use-package slim-mode
     :defer t))
 
+(defun html/init-jade-mode ()
+  (use-package jade-mode
+    :defer t
+    :config (add-hook 'jade-mode-hook #'highlight-indentation-mode)))
```

Comment:
Perhaps not -- Can make it a config setting if you really want me to -- It will take me max a few minutes to do it :smile_cat: